lib/Modules/OVSCharmUtils/OVSCharmUtils.psm1

```diff
@@ -281,47 +293,66 @@ function Disable-OVS {
             $ovsRunning = $false
         }
     }
-
     if($ovsRunning) {
         $bridges = Start-ExternalCommand { & $OVS_VSCTL list-br }
         foreach($bridge in $bridges) {
             Start-ExternalCommand { & $OVS_VSCTL del-br $bridge }
         }
     }
-
     foreach($svcName in $ovsServices) {
         $svc = Get-Service -Name $svcName -ErrorAction SilentlyContinue
         if($svc) {
             Stop-Service $svcName -Force
             Disable-Service $svcName
         }
     }
-
     Disable-OVSExtension
 }
 
 function Enable-OVS {
     Enable-OVSExtension
-
     $ovsServices = @($OVS_OVSDB_SERVICE_NAME, $OVS_VSWITCHD_SERVICE_NAME)
     foreach($svcName in $ovsServices) {
         Enable-Service $svcName
         Start-Service $svcName
     }
+    Invoke-JujuCommand -Command @($OVS_VSCTL, 'set-manager', 'ptcp:6640:127.0.0.1')
+}
+
+function New-OVSWindowsServices {
+    $ovsSvc = Get-ManagementObject -ClassName Win32_Service -Filter "Name='$OVS_VSWITCHD_SERVICE_NAME'"
+    $ovsDBSvc = Get-ManagementObject -ClassName Win32_Service -Filter "Name='$OVS_OVSDB_SERVICE_NAME'"
+    if(!$ovsSvc -and !$ovsDBSvc) {
+        Write-JujuWarning "OVS services are not created yet"
+        return
+    }
+    if(($ovsSvc.PathName -eq $OVS_SERVICES[$OVS_VSWITCHD_SERVICE_NAME]['binary_path']) -and
+       ($ovsDBSvc.PathName -eq $OVS_SERVICES[$OVS_OVSDB_SERVICE_NAME]['binary_path'])) {
+        Write-JujuWarning "OVS services are correctly configured"
+        return
+    }
+    Stop-Service $OVS_VSWITCHD_SERVICE_NAME
+    Stop-Service $OVS_OVSDB_SERVICE_NAME -Force
+    Remove-WindowsServices -Names @($OVS_VSWITCHD_SERVICE_NAME, $OVS_OVSDB_SERVICE_NAME)
```

Comment:
Yep, but I needed to add an extra parameter to the binary executed by the `$OVS_OVSDB_SERVICE_NAME` in order to allow me to set the OVS manager and thus use the native OVS interface for the neutron agent.